packages/eslint-plugin-pf-codemods/lib/rules/nav-list-variant.js

```diff
@@ -0,0 +1,39 @@
+const { getPackageImports } = require('../helpers');
+
+// https://github.com/patternfly/patternfly-react/pull/4225
+module.exports = {
+  create: function(context) {
+    const imports = getPackageImports(context, '@patternfly/react-core')
+        .filter(specifier => ['NavList', 'Nav'].includes(specifier.imported.name));
+    const navImport = imports.find(imp => imp.imported.name === 'Nav');
+    const NavListImport = imports.find(imp => imp.imported.name === 'NavList');
+
+    return !navImport || !NavListImport ? {} : {
+      JSXElement(node) {
+        if (NavListImport.local.name === node.openingElement.name.name) {
+          const hasNavParent = node.parent
+              && node.parent.openingElement.name.name === navImport.local.name
```

Comment:
will probably error in this case because it is not wrapped in a parent node
File 1: 
```
import { Nav } from '@patternfly/react-core';
import { MyListComponent } from './myList';

<Nav><MyListComponent /></Nav>
```
myList.tsx:
```
import { NavList } from '@patternfly/react-core';

<NavList variant="horizontal" />
```

Reply:
Done